### PR TITLE
Upgrade Twig to 2.x

### PR DESCRIPTION
**Pull request by thijskh**

### CODE REVIEW

**Comment by thijskh** [comment]
The testsuite fails only on the security audit which highlights a vulnerability irrelevant to EB...

File: theme/base/templates/layouts/scripts/default.html.twig [hunk 1]
```diff
@@ -14,7 +14,7 @@
         <link rel="preload" href="/fonts/sourcesanspro-bold-webfont.woff2?v={{ assetsVersion }}" as="font" type="font/woff2" crossorigin>
         <link rel="preload" href="/fonts/nunito-semibold.woff2?v={{ assetsVersion }}" as="font" type="font/woff2" crossorigin>
         <link rel="preload" href="/fonts/sourcesanspro-regular-webfont.woff2?v={{ assetsVersion }}" as="font" type="font/woff2" crossorigin>
-        <link href="/stylesheets/application.css?v={{ assetsVersion }}" rel="stylesheet" type="text/css"/>
+        <link href="/stylesheets/application.css?v={{ assetsVersion }}" rel="stylesheet">
```
Comment (MKodde) [inline theme/base/templates/layouts/scripts/default.html.twig:17]: On multiple occasions you removed the `type` attribute from the `link` elements. No problem, all browsers will implement this correctly and will automatically assume a stylesheet is of type text/css. 

I wonder why you removed this explicit type definition.

File: theme/base/templates/modules/Default/Partials/notConverted/footer.html.twig [hunk 2]
```diff
@@ -1,4 +1,6 @@
+{% import "@themeLayouts/scripts/language-switcher.html.twig" as languageSwitcher %}
```
Comment (MKodde) [inline theme/base/templates/modules/Default/Partials/notConverted/footer.html.twig:1]: Nice move to load the language switcher only in the templates where it is actually used :+1:

**Review by MKodde — APPROVED** [review]
Thanks for updating Twig to a modern version! See two points in the code review below.

File: theme/base/templates/layouts/scripts/default.html.twig (hunk 1 above)
Comment (thijskh) [reply theme/base/templates/layouts/scripts/default.html.twig:17]: It is considered [recommended practice in HTML5 to leave it out](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link#attr-type).

File: theme/base/templates/modules/Default/Partials/notConverted/footer.html.twig (hunk 2 above)
Comment (thijskh) [reply theme/base/templates/modules/Default/Partials/notConverted/footer.html.twig:1]: It's actually required in Twig 2. Importing a macro makes it only available to the exact template file it's imported in, not child/included templates.

**Pull request merged**